person.rb

```diff
@@ -0,0 +1,45 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission = 1)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getter methods
+  def getid
+    @id
+  end
+
+  def getname
+    @name
+  end
+
+  def getage
+    @age
+  end
+
+  # setter methods
+  def setname=(value)
+    @name = value
+  end
+
+  def setage=(value)
+    @age = value
+  end
```

Comment:
- [ ] You already have specified an `attr_reader` for `id` which is a getter method and an `attr_accessor` for `name` and `age` variables which is a combination of a getter and a setter. There is no need to specify custom setter and getter methods if we are not doing something extra in them. Please remove these redundant methods. 🧐️